package.json

```diff
@@ -64,4 +64,4 @@
     "set-value": "^2.0.1",
     "yargs-parser": "^13.1.2"
   }
-}
```

Comment:
Same as other comment on minor changes - guessing this is caused by auto formatting.

Reply:
removed